jquery.flot.js

```diff
@@ -13,6 +13,26 @@ Licensed under the MIT license.
     var hasOwnProperty = Object.prototype.hasOwnProperty;
     var Canvas = window.Flot.Canvas;
 
+    var saturated = {
```

Comment:
functions to limit a number to the double range, trying to prevent +Inf/-Inf being generated in computations